app/screens/attachment_options/index.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {StyleSheet} from 'react-native';
+
+import BottomSheet from '@screens/bottom_sheet';
+
+import AttachmentOptionContainer from './attachment_option_container';
+
+import type {AttachmentOptionsProps} from '@app/components/post_draft/quick_actions/attachment_options/attachment_options.types';
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+type Props = {
+    componentId: AvailableScreens;
+    closeButtonId: string;
+} & AttachmentOptionsProps;
+
+const style = StyleSheet.create({
+    contentStyle: {
+        paddingTop: 14,
+    },
+});
+
+const AttachmentOptionsScreen: React.FC<Props> = ({
+    componentId,
+    closeButtonId,
+    canUploadFiles,
+    fileCount,
+    maxFileCount,
+    onUploadFiles,
+    maxFilesReached,
+    testID,
+}) => {
+    const renderComponent = () => {
```

Comment:
This is passed as a prop, so use `useCallback`.

Reply:
Sure.